docs/quarantine.md

```diff
@@ -102,3 +102,27 @@ After merging this PR the test will be out of quarantine.
 [2]: https://www.thoughtworks.com/en-us/insights/blog/no-more-flaky-tests-go-team
 [3]: https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html#quarantined-tests
 [on testgrid]: https://testgrid.k8s.io/kubevirt-periodics
+
+# Test Lane Quarantine
+
+There can be cases where required test lanes are flaking in a clustered manner with a large
+number of tests failing each time. This can cause major delays to merging important pull
+requests and overload CI. Individual test case quarantining does not make sense in this case.
+
+A required test lane should be made optional if the following criteria are met:
+* The required test lane has a failure rate higher than 40% in the last 24 hours
```

Comment:
Do we have a clear metric to measure the failure rate?

IMHO 40% is way too high. What is the current failure rate of our worst lane? I don't think we should let a lane that is failing 10% of the time to block the rest of the project.

Reply:
> Do we have a clear metric to measure the failure rate?

We can search for the flake and see the percentage impact of that flake - for example - https://search.ci.kubevirt.io/?search=log+line+even+trying+to+flood+the+serial+console+for+QOSGuaranteed+VMs&maxAge=336h&context=1&type=bug%2Bissue%2Bjunit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

> 
> IMHO 40% is way too high. What is the current failure rate of our worst lane? I don't think we should let a lane that is failing 10% of the time to block the rest of the project.

Yes I think it is a bit high too - I was working off old numbers in my head where we used to have an average failure rate of about 20% - we are now averaging between ~ 9% - 13% failure rate overall e2e lanes[1]. 25% failure rate is a better limit as that is when it will begin to impact our ability to merge PRs. 

[1]https://grafana.ci.kubevirt.io/d/efpTS3t4z/e2e-jobs-overview-v2?orgId=1&viewPanel=13&from=1722294422579&to=1723452234415

Reply:
Thanks. Would you include a short explanation how I can tell what is the failure rate of a specific lane, or how I can find those lanes that fail more often than 25%?
It would help me, and I think also others, understand when it's time to downgrade a lane.

Reply:
Just to understand, now individual tests are failing and they make up to 20% of failure rate for a lane, right? So what do we want to target here?